js/src/utils/isMobileApp.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Check if the WC app is running on iOS.
+ *
+ * @return {boolean} Whether the WC app is running on iOS.
+ *
+ */
+const isWCIos = () => {
+	return window.navigator.userAgent.includes( 'wc-ios' );
```

Comment:
Don't think this should be a problem, but in the PHP solution a non case sensitive solution was used, this one is case sensitive. If that's acceptable then ok to leave as is.